.github/workflows/tests.yaml

```diff
@@ -4,12 +4,15 @@ on:
   push:
     branches: [main, develop, test-actions]
   pull_request:
-    branches: [main, develop, test-actions]
```

Comment:
Any particular reason to remove this here but not in the docker actions?

Reply:
No, I'll make that change.

Reply:
Oh right, I only now realised this only concerns pull requests. That makes sense.